src/patient-registration/input/custom-input/select-input.component.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
```

Comment:
This file should be deleted once [MF-285](https://issues.openmrs.org/browse/MF-285) has been completed to fix the `SelectInput` component.

Reply:
Will be waiting on this...

Reply:
I'll be working on it today and will hopefully have an fix in place by the end of today.

Reply:
Cool....

Reply:
Will let you know as soon as it's done

Reply:
[PR](https://github.com/openmrs/openmrs-esm-patient-registration/pull/24)

Reply:
Thanks for the PR...The challenge with using this generic select input is that the name it renders would appear as Relationship[0].type since we are using a fieldArray(compare [this](https://github.com/openmrs/openmrs-esm-patient-registration/pull/24/files#diff-9d727965793991108ca83b53ee5b44edL14) and [this which I hard coded](https://github.com/openmrs/openmrs-esm-patient-registration/pull/20/files#diff-2d15cdc8a1735e458c25f6fcdd96a82dR14).

The second challenge is that the value and display are different and not the same as [it is used here](https://github.com/openmrs/openmrs-esm-patient-registration/pull/24/files#diff-9d727965793991108ca83b53ee5b44edR17-R18) in this case we need the option.uuid as value while the display as the option.display [as seen here](https://github.com/openmrs/openmrs-esm-patient-registration/pull/20/files#diff-2d15cdc8a1735e458c25f6fcdd96a82dR17-R18)